src/main/java/org/checkerframework/checker/crypto/CryptoAnnotatedTypeFactory.java

```diff
@@ -90,5 +155,12 @@ private boolean compareAllowedAlgorithmOrProviderTypes(
                     AnnotationUtils.getElementValueArray(subtype, "value", String.class, true);
             return supertypeRegexList.containsAll(subtypeRegexList);
         }
+
+        private AnnotationMirror createCryptoAnnotation(
```

Comment:
Some documentation would be good. I would swap the order of the parameters - you're building an annoClass with the values, so that order seems more natural to me.